javascriptv3/example_code/medical-imaging/tests/create-delete-datastore.integration.test.js

```diff
@@ -0,0 +1,38 @@
+import {describe, it, expect} from "vitest";
+
+import {createDatastore} from "../actions/create-datastore.js";
+import {deleteDatastore} from "../actions/delete-datastore.js";
+
+const sleep = (delay) => new Promise((resolve) => setTimeout(resolve, delay))
+describe("createDatastore/deleteDatastore", () => {
+        let datastoreID = "";
+        const datastoreName = "createDeleteDatastoreTest";
+
+        it("should create and delete a datastore", async () => {
+            // Create topic.
+            const createDatastoreCommandOutput = await createDatastore(datastoreName);
+            datastoreID = createDatastoreCommandOutput.datastoreId;
+
+            expect(datastoreID).toBeDefined();
+
+            await sleep(10000); // Wait for the datastore to be created.
```

Comment:
Is there an alternative to this arbitrary wait?

I have a `wait` and `retry` function in `example_code/libs/utils/util-timers.js` if that helps. `retry` will keep trying an async function over a period of time if it fails.